src/server/game/Entities/Player/Player.cpp

```diff
@@ -18742,9 +18742,34 @@ bool Player::LoadFromDB(ObjectGuid guid, CharacterDatabaseQueryHolder* holder)
 
     m_achievementMgr->CheckAllAchievementCriteria(this);
     m_questObjectiveCriteriaMgr->CheckAllQuestObjectiveCriteria(this);
+
+    _PushQuests();
```

Comment:
Any other cases when quests are pushed to players? Levelup? Zone change?

Reply:
I added zone change before I saw this, I forgoet levelup though. I'll do that too.